src/wodan/wodan.mli

```diff
@@ -15,15 +15,18 @@
 (*                                                                              *)
 (********************************************************************************)
 
-val max_dirty : int
+(** The Wodan module allows access to Wodan filesystems
+
+    The entry point is {!Wodan.Make}, which returns a
+    {!Wodan.S} module supporting filesystem operations *)
 
 exception BadMagic
 
 exception BadVersion
 
 exception BadFlags
 
-exception BadCRC of Location.t
+exception BadCRC of int64
```

Comment:
What is the rationale here? 
It seems to me like `Location.t` was a good abstraction here?

Reply:
It's a good abstraction, but it's not exposed externally; the exception leaked a module that wasn't intended to be visible.

Reply:
Ah indeed. Perhaps it deserves a docstring then, saying this is indeed a location (and not e.g. a CRC code).

Reply:
Added, along with docs for the other exceptions